.github/workflows/storybook-tests.yml

```diff
@@ -8,17 +8,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
-          node-version: "14.x"
+          node-version-file: ".node-version"
```

Comment:
Not strictly related to accessibility testing, but I noticed that tests are running on Node 14, but local dev is on Node 16.

Updated this to ensure consistency.

Reply:
Awesome, thank you 👍🏻